bindings/wasm/lib/worker.ts

```diff
@@ -58,18 +59,29 @@ export namespace MessageToWorker {
   export interface Initialize extends Message {
     type: 'initialize';
     manifoldWasmUrl?: string;
-    remotePackagePrefix?: string;
+    esbuildWasmUrl?: string;
   }
 
   /**
    * Evaluate a ManifoldCAD script.
    *
    * The worker will respond with `MessageFromWorker.Done`
    * or `MessageFromWorker.Error`.
+   *
+   * `filename` doesn't have much meaning at the moment, but it
+   * is useful for ensuring effective error messages later.
+   *
+   * If `bundle` is set to false, the worker will not bundle code,
+   * and assume that it has already been bundled.  An undefined
+   * value will be treated by default as true.
```

Comment:
`bundle` is ambiguous - it could mean "please bundle" or "is already a bundle". Maybe rename to `doBundle` or `applyBundling`?